containers/proxy-httpd/uyuni-configure.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+
+import os
+import yaml
+
+config_path = "/etc/uyuni/"
+
+# read from file
+with open(config_path + "config.yaml") as source:
+    config = yaml.safe_load(source)
+    
+    # Save the systemid file
+    with open("/etc/sysconfig/rhn/systemid", "w") as file:
+        file.write(str(config['system_id']))
+    
+    # Decode and install SSL CA certificate
+    os.system(f"echo -n '{config['binary_ca_certs']}' | base64 -d > /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT")
+    os.symlink("/etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT", "/usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT")
+    os.system("/usr/sbin/update-ca-certificates")
+
+    # Decode and install server certificate package
+    os.system(f"echo -n '{config['binary_srv_cert_rpm']}' | base64 -d > /root/rhn-org-httpd-ssl-key-pair-proxy-1.0-3.noarch.rpm")
+    os.system('rpm -Uv /root/rhn-org-httpd-ssl-key-pair-proxy-1.0-3.noarch.rpm')
```

Comment:
What does this RPM contain? Do we need the content to be RPM packaged at all?

Reply:
The package contains:

- `/etc/apache2/ssl.crt/server.crt`
- `/etc/apache2/ssl.csr/server.csr`
- `/etc/apache2/ssl.key/server.key`
- `/etc/pki/spacewalk/jabberd/server.pem`

The last one is useless so I dropped it. I also refactored to get rid of the rpm and we now have 3 parameters in the `config.yaml` with the encoded value of each file, to be decoded and used in the `uyuni-configure.py` script.